config/_default/menus/menus.en.yaml

```diff
@@ -1788,36 +1788,100 @@ main:
+    parent: security_monitoring
+    identifier: security_monitoring_getting_started
+    weight: 101
+  - name: Security Home
+    url: security_platform/security_monitoring/security_home
+    parent: security_monitoring
+    identifier: security_monitoring_security_home
+    weight: 102
+  - name: Log Detection Rules
+    url: security_platform/security_monitoring/log_detection_rules
+    parent: security_monitoring
+    identifier: security_monitoring_log_detection_rules
+    weight: 103
+  - name: Default Rules
+    url: /security_platform/default_rules#cat-log-detection
+    parent: security_monitoring
+    identifier: security_monitoring_default_rules
+    weight: 104
+  - name: Signals Explorer
+    url: /security_platform/explorer
+    parent: security_monitoring
+    identifier: security_monitoring_signals_explorer
+    weight: 105
+  - name: Cloud Security Posture Management
+    url: security_platform/cspm/
     parent: security_platform
-    identifier: security_rules
+    identifier: cspm
     weight: 2
-  - name: Security Monitoring
-    url: security_platform/detection_rules/security_monitoring
-    parent: security_rules
-    identifier: security_monitoring_detection_rules
+  - name: Getting Started
+    url: security_platform/cspm/getting_started
+    parent: cspm
+    identifier: cspm_getting_started
     weight: 201
+  - name: Frameworks and Benchmarks
+    url: security_platform/cspm/frameworks_and_benchmarks
+    parent: cspm
+    weight: 202
   - name: Default Rules
-    url: security_platform/default_rules
+    url: security_platform/cspm/configuration_rules
+    parent: cspm
+    identifier: cspm_default_rules
+    weight: 203
+  - name: Posture Findings
+    url: /security_platform/cspm/findings
+    parent: cspm
+    identifier: cspm_findings_explorer
+    weight: 204
+  - name: Cloud Workload Security
+    url: security_platform/cloud_workload_security/
     parent: security_platform
-    identifier: default_rules
+    identifier: cloud_workload_security
     weight: 3
+  - name: Getting Started
+    url: security_platform/cloud_workload_security/getting_started
+    parent: cloud_workload_security
+    identifier: cws_getting_started
+    weight: 301
+  - name: Workload Security Rules
+    url: security_platform/cloud_workload_security/workload_security_rules
+    parent: cloud_workload_security
+    identifier: cws_workload_security_rules
+    weight: 302
+  - name: Default Rules
+    url: security_platform/default_rules#cat-runtime-agent
+    parent: cloud_workload_security
+    identifier: cws_workload_default_rules
+    weight: 303
+  - name: Signals Explorer
+    url: security_platform/explorer
+    parent: cloud_workload_security
+    identifier: cws_workload_signals_explorer
+    weight: 304
+  - name: Security Rules
+    url: security_platform/detection_rules/
+    parent: security_platform
+    identifier: security_rules
+    weight: 4
   - name: Signals Explorer
```

Comment:
I think since you have it here, you don't need it in the two places in the toc above